tests/unit_tests/agent/node/test_schema_linking_node.py

```
@@ -0,0 +1,569 @@
+
+from datus.agent.node.schema_linking_node import SchemaLinkingNode
+from datus.schemas.action_history import ActionStatus
+from datus.schemas.node_models import TableSchema, TableValue
+from datus.schemas.schema_linking_node_models import SchemaLinkingInput, SchemaLinkingResult
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _make_agent_config(rag_path="/tmp/nonexistent_rag"):
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent_rag"

(S108)